olp-cpp-sdk-dataservice-read/test/unit/src/CatalogClientTest.cpp

```diff
@@ -3280,4 +3393,6 @@ TEST_P(CatalogClientCacheTest, GetVolatilePartitionsExpiry) {
   ASSERT_TRUE(partitionsResponse.IsSuccessful())
       << PrintError(partitionsResponse.GetError());
   ASSERT_EQ(0u, partitionsResponse.GetResult().GetPartitions().size());
+
+  EXPECT_TRUE(testing::Mock::VerifyAndClearExpectations(&*network_mock_));
```

Comment:
```suggestion
  testing::Mock::VerifyAndClearExpectations(network_mock_.get());
```

VerifyAndClearExpectations() already fails the test if some expects are not met. Wrapping this additionally in an expect is not necessary.